openapi/smartevents_mgmt.yaml

```diff
@@ -791,130 +800,207 @@ components:
         error_handler:
           $ref: '#/components/schemas/Action'
     BridgeResponse:
+      required:
+      - kind
+      - id
+      - href
+      - submitted_at
+      - status
+      - owner
+      - endpoint
       type: object
       properties:
         kind:
           type: string
+          nullable: false
         id:
           type: string
+          nullable: false
         name:
           type: string
         href:
           type: string
+          nullable: false
         submitted_at:
           format: date-time
           type: string
+          nullable: false
         published_at:
           format: date-time
           type: string
         status:
-          $ref: '#/components/schemas/ManagedResourceStatus'
+          allOf:
+          - $ref: '#/components/schemas/ManagedResourceStatus'
+          - nullable: false
         owner:
           type: string
+          nullable: false
         endpoint:
           type: string
+          nullable: false
```

Comment:
IIRC the bridge `endpoint` is only returned after the bridge has been provisioned. Like the `published_at` property.

Reply:
Good catch. I'll change it.